app/code/Magento/GiftMessage/Setup/InstallData.php

```diff
@@ -80,7 +80,8 @@ public function install(ModuleDataSetupInterface $setup, ModuleContextInterface
         /**
          * Add 'gift_message_available' attributes for entities
          */
-        $salesSetup->addAttribute('order_item', 'gift_message_available', $options);
+        $flagOptions = ['type' => \Magento\Framework\DB\Ddl\Table::TYPE_BOOLEAN, 'visible' => false, 'required' => false];
+        $salesSetup->addAttribute('order_item', 'gift_message_available', $flagOptions);
```

Comment:
This is not a refactoring. Please force push without the last commit.

Reply:
Done